opencsp/common/lib/geometry/Pxyz.py

```diff
@@ -20,3 +23,19 @@ def as_Vxyz(self):
     @classmethod
     def empty(cls):
         return Pxyz([[], [], []])
+
+    def draw_point(
+        self,
+        figure: rcfr.RenderControlFigureRecord | v3d.View3d,
+        style: rcps.RenderControlPointSeq = None,
+        labels: list[str] = None,
+    ):
+        """Calls figure.draw_xyz(p) for all points in this instance, and with
+        the default arguments in place for any None's."""
+        if style is None:
+            style = rcps.default(markersize=2)
+        if labels is None:
+            labels = [None] * len(self)
+        view = figure if isinstance(figure, v3d.View3d) else figure.view
+        for x, y, z, label in zip(self.x, self.y, self.z, labels):
+            view.draw_xyz((x, y, z), style, label)

```

Comment:
Would this make sense to add to Vxyz instead since Pxyz inherits Vxyz? 
Furthermore, what is the difference between `draw_point` and `draw_list`? Could they be combined into one function altogether?

Reply:
The difference is that draw_points draws each point individually, vs draw_list which draws all points together as a single series. Updated the documentation to match and moved the method into Vxyz.